rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-s1c.yaml

```diff
@@ -0,0 +1,25 @@
+---
+###############################################################################
+# SR-IOV Network Definition for the NS of the workload
+#  s1c-bh + vlan 803 + core_sriov_netdevice
+###############################################################################
+# NS = open5gs-core
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: s1cbh
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: core_sriov_netdevice
+  networkNamespace: open5gs-core
+  vlan: 803
+  ipam: |-
+    {
+      "type": "static",
+      "addresses": [
+        {
+          "address": "10.27.3.2/24"
```

Comment:
L21 this is bad, this is one of the router addresses (VRRP) in this particular subnet.  choose a different, higher IP address in this subnet please.

Reply:
Noted.